### PR TITLE
ru placetype local and more

### PR DESCRIPTION
**Pull request by nvkelso**
Will fix a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/2154. Sets placetype local, names, populations, and statoids properties for CARTO

### CODE REVIEW

File: data/110/873/259/5/1108732595.geojson [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
     "geom:latitude":52.625677,
     "geom:longitude":90.122365,
     "iso:country":"RU",
     "label:eng_x_preferred_longname":[
-        "Abaza District"
+        "Abaza Raion"
     ],
     "label:eng_x_preferred_placetype":[
-        "district"
+        "raion"
```
Comment (stepps00) [inline data/110/873/259/5/1108732595.geojson]: Similar to some of the name properties.. worth moving the existing "district" value to a `label:eng_x_variant_placetype` property? Same goes for any label / name prop we're updating.

File: data/110/873/260/3/1108732603.geojson [hunk 2]
```diff
@@ -5,16 +5,16 @@
...
     "geom:area":0.185123,
-    "geom:area_square_m":1622352379.128015,
+    "geom:area_square_m":1622352557.485719,
     "geom:bbox":"37.986778,44.581104,38.591133,45.190545",
     "geom:latitude":44.867684,
     "geom:longitude":38.283528,
     "iso:country":"RU",
     "label:eng_x_preferred_longname":[
-        "Abinskiy District"
+        "Abinskiy Raion"
```
Comment (stepps00) [inline data/110/873/260/3/1108732603.geojson]: This looks very similar to the `rus_latn` longname we have on this record, though the spelling is slightly different. Expected?

File: data/110/873/259/5/1108732595.geojson (hunk 1 above)
Comment (nvkelso) [reply data/110/873/259/5/1108732595.geojson]: Agree, I'll add a new commit...

File: data/110/873/277/3/1108732773.geojson [hunk 3]
```diff
@@ -5,16 +5,16 @@
...
     "geom:area":0.035035,
-    "geom:area_square_m":306664421.789024,
+    "geom:area_square_m":306664275.31873,
     "geom:bbox":"40.893594,44.776366,41.265029,45.064658",
     "geom:latitude":44.936456,
     "geom:longitude":41.110437,
     "iso:country":"RU",
     "label:eng_x_preferred_longname":[
-        "Armavir District"
+        "Armavir Gorsovet"
```
Comment (stepps00) [inline data/110/873/277/3/1108732773.geojson]: Diving into a couple examples, I see this place referred to as the City of Armavir - an administrative unit with the status equal to that of the districts. 

https://en.wikipedia.org/wiki/Armavir,_Russia

Just curious, where is the "Gorsovet" coming from?

**Review by stepps00 — CHANGES_REQUESTED** [review]
Looks great - a couple questions about replacing vs storing variants, one question about a specific feature.

File: data/110/873/260/3/1108732603.geojson (hunk 2 above)
Comment (nvkelso) [reply data/110/873/260/3/1108732603.geojson]: Ohhhh, those `rus_latn` longnames are useful!

File: data/110/873/277/3/1108732773.geojson (hunk 3 above)
Comment (nvkelso) [reply data/110/873/277/3/1108732773.geojson]: I had taken these new values from Statoids. But it looks like they are too coarse compared to other options in `meso:*`? But more importantly Statoids is a poor match for what's in WOF so some features were getting new Raion values, and some were remaining District leading to an inconsistent state where 1530 records were updated out of the 2508 counties in Russia.

This is too much to untangle right now so I'll reset to `District` and we can revisit later...

Says Statoids for [Russia](http://statoids.com/yru.html):

**T: Type of unit.**

```
g = gorod
n = natsional'nyy rayon
o = ostrov
p = pgt
r = raion
s = gorsovet
v = Natsional'naya Volost'
? = type unknown
```

Comment (nvkelso) [reply data/110/873/277/3/1108732773.geojson]: The only other widespread placetype label change was in Sudan for 172 records, which still seems appropriate (as long as the placetype label variant is also set).

Comment (nvkelso) [reply data/110/873/277/3/1108732773.geojson]: Reset back to `District` via https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/24/commits/66601ef140b14b95d1c32a7684b487fb2feee829.

File: data/110/873/260/3/1108732603.geojson (hunk 2 above)
Comment (nvkelso) [reply data/110/873/260/3/1108732603.geojson]: Reset back to `District` via https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/24/commits/66601ef140b14b95d1c32a7684b487fb2feee829.

(Someday we can dive into all the different types of districts and add those as alt long names...)

File: data/110/873/259/5/1108732595.geojson (hunk 1 above)
Comment (nvkelso) [reply data/110/873/259/5/1108732595.geojson]: Reset back to `district` via https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/24/commits/66601ef140b14b95d1c32a7684b487fb2feee829.

File: data/110/873/277/3/1108732773.geojson (hunk 3 above)
Comment (stepps00) [reply data/110/873/277/3/1108732773.geojson]: Thanks for the detail, super interesting! These changes all check out...

File: data/110/873/260/3/1108732603.geojson (hunk 2 above)
Comment (stepps00) [reply data/110/873/260/3/1108732603.geojson]: looks good!

**Review by stepps00 — APPROVED** [review]
LGTM

**Pull request merged**